ngraph/core/include/ngraph/opsets/opset7_tbl.hpp

```diff
@@ -90,6 +90,7 @@ NGRAPH_OP(RegionYolo, ngraph::op::v0)
 NGRAPH_OP(ReorgYolo, ngraph::op::v0)
 NGRAPH_OP(Reshape, ngraph::op::v1)
 NGRAPH_OP(Result, ngraph::op::v0)
+NGRAPH_OP(Reverse, ngraph::op::v1)
```

Comment:
We should not include the Reverse to the opset7 since no plugin support it.

Reply:
Agree, opset7 have no such operator:
https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/opset7.md

Reply:
@lazarevevgeny 
I wonder why some of the previously failed tests now pass even if there is no plugin to handle it. 
Is there any transformation which remove it?

Reply:
If the inserted Reverse operation is located on a shapeOf sub-graph then it will be constant folded. 

Or the CPU plugin uses fallback to reference implementation of the operation when it does not support some operation.